Protos/fetch.sh

```diff
@@ -33,12 +33,21 @@ rm -rf "$upstream"
 # rather than source repository name.
 mkdir -p "$upstream/grpc"
 mkdir -p "$upstream/google"
+mkdir -p "$upstream/grpc/testing"
+mkdir -p "$upstream/grpc/core"
```

Comment:
nit, we no longer need to `mkdir grpc` if we make subdirectories within it:

```suggestion
mkdir -p "$upstream/google"
mkdir -p "$upstream/grpc/testing"
mkdir -p "$upstream/grpc/core"
```